aws-hybrid-vpn-gcp/inputs.tf

```diff
@@ -0,0 +1,30 @@
+variable "name" {
+  description = "Name of the Hybrid VPN deployment, used in various resources to uniquely identify them"
+  type        = string
+  default     = "test"
+}
+
+variable "aws_vpc_id" {
+  description = "AWS VPC to connect the VPN to"
+  type        = string
+}
+
+variable "aws_subnet_ids" {
+  description = "AWS subnets to connect the VPN to"
+  type        = list(string)
+}
+
+variable "gcp_network_name" {
```

Comment:
see next comment re: `vpc` vs `network`

Reply:
For consistency with the variables that are used.
https://github.com/honestbank/terraform-gcp-vpc/blob/main/outputs.tf#L1-L4